test/e2e/data/kubetest/conformance-fast.yaml

```diff
@@ -5,3 +5,4 @@ ginkgo.progress: true
 ginkgo.slowSpecThreshold: 120.0
 ginkgo.trace: true
 ginkgo.v: true
+ginkgo.timeout: 10800s
```

Comment:
Good question, I think so

> -timeout=24h (or some other, suitable value) must be passed because the default timeout was reduced from 24h to 1h

let's try

Reply:
changed to 3h